cfgov/scripts/http_smoke_test.py

```diff
@@ -119,7 +119,7 @@
     "/paying-for-college/",
     "/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/",  # noqa: E501
     "/retirement/before-you-claim/",
-    "/retirement/before-you-claim/es/",
+    "/es/retirement/before-you-claim/",
```

Comment:
Shouldn't this be /es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/? Why do both https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/es/ and https://www.consumerfinance.gov/es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/ still work?

Reply:
Fixed in d35096f (plus a bonus fix to a url I missed)